cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +50,65 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it("Switch from 'Webhook' to 'Ignore' Error handler", () => {
+      //Open 'Instance one' page
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .within(() => {
+          cy.get(
+            "td:first a[data-testid='tableInstances-linkInstance']"
+          ).click();
+        });
+
+      // Switch to error handling tab
+      cy.ouiaId("error-handling", "PF4/TabButton").click();
+
+      // Wait tab is loaded
+      cy.get(".pf-c-description-list__term").should("have.length", 5);
+      // Once tab is loaded start edit mode
+      cy.ouiaId("error-handling", "PF4/TabContent").within(() => {
+        cy.ouiaId("edit", "PF4/Button")
+          .should("have.attr", "aria-disabled")
+          .and("equal", "false");
+        cy.ouiaId("edit", "PF4/Button").click();
+      });
+
+      // Change error handling strategy to 'Ignore'
+      cy.ouiaId("error-handling-method-selector", "PF4/Select").within(() => {
+        cy.get("button").should("not.have.attr", "disabled");
+      });
+      cy.ouiaId("error-handling-method-selector", "PF4/Select").click();
+      cy.get("button.pf-c-select__menu-item", { timeout: 10000 })
+        .contains("Ignore")
+        .click();
+      cy.ouiaId("submit", "PF4/Button").click();
+
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .should("be.visible")
+        .within(() => {
+          cy.get("td:first").should("contain.text", "Instance one");
+          //The status column is loading and contains ready
+          cy.get("td:nth-child(3)")
+            .ouiaId("accepted", "QE/ResourceStatus")
+            .within(() => {
+              cy.get("span[role='progressbar']").should("exist");
+              cy.ouiaId("creating", "PF4/Button").should(
+                "have.text",
+                "Creating"
+              );
+              cy.ouiaId("ready-shortly", "QE/HelperTextItem").should(
+                "have.text",
+                "This will be ready shortly."
+              );
+            });
+          cy.ouiaId("creating", "PF4/Button").click();
```

Comment:
Do we need to test Popover here? Is possible to check the `ouiaId("ready", "QE/ResourceStatus")` status

Reply:
thank you for a comment. is add5500 what you had in mind @sjamboro ?